backtracking/graph_coloring.cpp

```diff
@@ -95,7 +95,7 @@ int main() {
     };
     int m = 3;  // Number of colors
 
-    int color[V];
+    std::array <int, V> color = {};
```

Comment:
`std::array` does not support dynamic arrays. You will have to use pointers to `std::vector`